content/articles/email-validation-in-flutter/index.md

```diff
@@ -0,0 +1,181 @@
+
+In this article, we are going to learn how to validate emails using an email validator. We are going to develop a simple flutter application to show this.
+
+  
+
+### Introduction
+
+The main motive of this article is to learn how to validate emails in Flutter applications. The mail will be validated for both serverless app and those that are server enabled. It means there will not be any backend or database for the service, it will work on the device itself. We are going to parse the email at the client's end and then pass it forward for validation.
+
+For this example, We are using Flutter which provides the email validator package. Email validator is a Dart class for validating email addresses without using Reg Ex. Whenever there is some exception or validation fails, it will notify you on the screen with an alert view. It also shows how many times that particular mail(email) has failed in past attempts.
+
+### Prerequisite
```

Comment:
Add relevant links to some of these prerequisites.

Reply:
@Jethro-magaji. I was to make the correction immediately after you requested for them. Kindly reopen I make a commit of the revised instead of a new pr